strategies/indicators/DEMA.js

```diff
@@ -1,27 +1,22 @@
 // required indicators
 var EMA = require('./EMA.js');
+var SMA = require('./SMA.js');
 
 var Indicator = function(config) {
-  this.input = 'price'	
+  this.input = 'price'
   this.result = false;
-  this.short = new EMA(config.short);
-  this.long = new EMA(config.long);
+  this.inner = new EMA(config.weight);
+  this.outer = new EMA(config.weight);
+  this.sma = new SMA(config.weight); // It will be used in the strategy
```

Comment:
Indicator should not include any code needed for a strategy. How should I suppose to use this indicator later in my own strategies? 0 reusability